Wrapping/CSharp/CSharp.i

```diff
@@ -43,6 +43,24 @@
 %CSharpTypemapHelper( float*, System.IntPtr )
 %CSharpTypemapHelper( double*, System.IntPtr )
 
+
+// CSharp does not hadle overloaded const methods. So they are
+// explicitly renamed to enable their explicit usage. These methods
+// assume that the buffer is will not be modified, so MakeUnique is
+// not call.
+%rename("GetConstBufferAsInt8")   itk::simple::Image::GetBufferAsInt8( ) const;
```

Comment:
// assume that the buffer is will not be modified, so MakeUnique is
// not call.

change to:
// assume that the buffer will not be modified, so MakeUnique is
// not called.